sergii.holubenko/task_1.rb

```diff
@@ -0,0 +1,25 @@
+arr = [a, b, c]
```

Comment:
it won't work, you will see an error `NameError (undefined local variable or method `a' for main:Object)` because here not defined a, b and c variables, yet